pkg/actors/actors.go

```diff
@@ -800,25 +803,32 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 		RegisteredTime: time.Now().UTC().Format(time.RFC3339),
 	}
 
-	reminders, err := a.getRemindersForActorType(req.ActorType)
-	if err != nil {
-		return err
-	}
+	err := backoff.Retry(func() error {
```

Comment:
@artursouza qq should a max retries number be set to avoid the operation being tried again and again?

Reply:
I am leaving the defaults, which stops after 15 minutes:
 	DefaultMaxElapsedTime      = 15 * time.Minute

https://github.com/cenkalti/backoff/blob/c2975ffa541a1caeca5f76c396cb8c3e7b3bb5f8/exponential.go#L80